pkg/koordlet/util/node.go

```diff
@@ -58,6 +66,23 @@ func GetRootCgroupCurCPUSet(qosClass corev1.PodQOSClass) ([]int32, error) {
 	return cpuset.ParseCPUSet(cpus), nil
 }
 
+// GetBeCgroupCurCPUSet gets the current cpuset of besteffort podQoS' cgroup.
+func GetBeCgroupCurCPUSet() ([]int32, error) {
+	containerPaths, err := GetBECPUSetPathsByTargetDepth(ContainerCgroupPathRelativeDepth)
+	if err != nil {
+		return nil, err
+	}
+	if len(containerPaths) == 0 {
+		return GetRootCgroupCurCPUSet(corev1.PodQOSBestEffort)
```

Comment:
Since the `GetRootCgroupCurCPUSet()` and `GetKubeQosRelativePath()` are DEPRECATED, please try using `GetPodQoSRelativePath(corev1.PodQOSBestEffort)` instead to get the target cgroup path.